content/en/blog/_posts/XXXX-XX-XX-kubernetes-1-34-sa-tokens-image-pulls-beta.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: blog
+title: "Kubernetes v1.34: Service Account Token Integration for Image Pulls Graduates to Beta"
+date: XXXX-XX-XX
+slug: kubernetes-v1-34-sa-tokens-image-pulls-beta
+draft: true
+author: >
+  [Anish Ramasekar](https://github.com/aramase) (Microsoft)
+---
+
+The Kubernetes community continues to advance security best practices
+by reducing reliance on long-lived credentials.
+Following the successful [alpha release in Kubernetes v1.33](/blog/2025/05/07/kubernetes-v1-33-wi-for-image-pulls/),
+**Service Account Token Integration for Kubelet Credential Providers**
+has now graduated to **beta** in Kubernetes v1.34,
+bringing us closer to eliminating long-lived image pull secrets from Kubernetes clusters.
+
+This enhancement allows credential providers
+to use workload-specific service account tokens to obtain registry credentials,
+providing a secure, ephemeral alternative to traditional image pull secrets.
+
+## What's new in beta?
+
+The beta graduation brings several important changes
+that make the feature more robust and production-ready:
+
+### Required `cacheType` field
+
+**Breaking change from alpha**: The `cacheType` field is **required**
+in the credential provider configuration when using service account tokens.
+This field is new in beta and must be specified to ensure proper caching behavior.
+
+```yaml
+tokenAttributes:
+  serviceAccountTokenAudience: "my-registry-audience"
+  cacheType: "ServiceAccount"  # Required field in beta
+  requireServiceAccount: true
+```
+
+Choose between two caching strategies:
+- **`Token`**: Cache credentials per service account token
+  (use when credential lifetime is tied to the token)
```

Comment:
Describe this in a bit more detail, i.e. this is primarily useful for registries that directly support image pulls using the KSA token directly.

Reply:
updated!